static/js/invite.js

```diff
@@ -99,6 +99,8 @@ function submit_invitation_form() {
         },
         complete() {
             $("#submit-invitation").button("reset");
+            $("#invitee_emails").trigger("focus");
+            document.getElementById("invite_status").scrollIntoView();
```

Comment:
This works fine, but I think this may lead to errors when the "invite_status" div is empty. The jquery scrollTop() did not work here for some reason. Please suggest corrections/changes.

Reply:
Well `$("div")` selects both, you need to use a more specific selector.